### DISCUSSION TITLE
"No 'borg' directory in the given backup directory found!"

### DISCUSSION THREAD

**Discussion by Irrigate6991** [nextcloud/all-in-one#525] [Questions] [answered]
"No 'borg' directory in the given backup directory found!
Please adjust the directory so that the borg archive is positioned in a folder named 'borg' inside the given directory!
Repository /mnt/borgbackup/borg does not exist."

Any ideas?
I tried "/mnt/borgbackup/borg", "/mnt/borgbackup/" and "/mnt/backup"

![Screenshot 2022-04-24 023214](https://user-images.githubusercontent.com/23613906/164950797-84e1de9e-a3d7-4969-8c66-e391373960f7.jpeg)

**Comment by szaimen** [comment]
Hi, did you solve it?

**Reply by Irrigate6991** [reply]
Hi, no

**Reply by szaimen** [reply]
The reworked process is available since https://github.com/nextcloud/all-in-one/releases/tag/v1.1.0

**Comment by szaimen** [comment, marked as answer]
We'll make the whole process better understandable with https://github.com/nextcloud/all-in-one/pull/527

**Reply by szaimen** [reply]
The reworked process is available since https://github.com/nextcloud/all-in-one/releases/tag/v1.1.0
